x/superfluid/keeper/internal/events/emit_test.go

```diff
@@ -156,6 +156,50 @@ func (suite *SuperfluidEventsTestSuite) TestEmitSuperfluidDelegateEvent() {
 	}
 }
 
+func (suite *SuperfluidEventsTestSuite) TestEmitCreateFullRangePositionAndSuperfluidDelegateEvent() {
+	testcases := map[string]struct {
+		ctx     sdk.Context
+		lockID  uint64
+		valAddr string
+	}{
+		"basic valid": {
+			ctx:     suite.CreateTestContext(),
+			lockID:  1,
+			valAddr: sdk.AccAddress([]byte(addressString)).String(),
+		},
+		"context with no event manager": {
+			ctx: sdk.Context{},
+		},
+	}
+
+	for name, tc := range testcases {
+		suite.Run(name, func() {
+			expectedEvents := sdk.Events{
+				sdk.NewEvent(
+					types.TypeEvtSuperfluidDelegate,
+					sdk.NewAttribute(types.AttributeLockId, fmt.Sprintf("%d", tc.lockID)),
+					sdk.NewAttribute(types.AttributeValidator, tc.valAddr),
+				),
+			}
```

Comment:
Can we add position ID to this test please

Reply:
Added :)